src/redux/data.js

```diff
@@ -429,6 +432,16 @@ export const dataWatchPendingTransactions = () => async (
   return false;
 };
 
+const updatePurchases = updatedTransactions => dispatch => {
+  const confirmedPurchases = filter(updatedTransactions, txn => {
+    return (
+      txn.type === TransactionTypes.purchase &&
+      txn.status !== TransactionStatusTypes.purchasing
```

Comment:
I'm wondering if we could do `txn.status === TransactionStatusTypes.purchased` instead

Reply:
I also wanted to include anything with TransactionStatusTypes.failed (but only if it was of a purchase type).